system/Session/Session.php

```diff
@@ -317,35 +379,35 @@ public function set($data, $value = null)
 	//--------------------------------------------------------------------
 
 	/**
-	 * Get any user data that has been set in the session.
+	 * Get user data that has been set in the session.
 	 *
+	 * If the property exists as "normal", returns it.
+	 * Otherwise, returns an array of any temp or flash data values with the
+	 * property key.
+	 * 
 	 * Replaces the legacy method $session->userdata();
 	 *
-	 * @param null $key
-	 *
-	 * @return array|null
+	 * @param  $key	Identifier of the session property to retrieve
+	 * @return array|null	The property value(s)
 	 */
 	public function get($key = null)
 	{
 		if (isset($key))
 		{
 			return isset($_SESSION[$key]) ? $_SESSION[$key] : null;
-		}
-		elseif (empty($_SESSION))
+		} elseif (empty($_SESSION))
```

Comment:
Just thought I'd chime in that I prefer `elseif`. I think I picked it up when I started using PSR-2. I sometimes find `else if` ambiguous in intent (did they mean `elseif` or `else { if `?), but that's not as much of an issue if everyone is good about always using braces (`{ ... }`) in their if/else statements.